invenio_rdm_records/assets/semantic-ui/js/invenio_rdm_records/src/deposit/api/DepositApiClient.js

```diff
@@ -41,7 +41,7 @@ export class DepositApiClient {
       withCredentials: true,
       xsrfCookieName: "csrftoken",
       xsrfHeaderName: "X-CSRFToken",
-      headers: this.apiHeaders.json,
+      headers: this.apiHeaders["vnd+json"],
```

Comment:
this change assumes vnd+json is defined

Reply:
just noticed it was like this before but below, still not sure what are the consequences

Reply:
Doesn't the same apply for `.json`? Both are defined [in `BASE_HEADERS`](https://github.com/inveniosoftware/invenio-rdm-records/blob/76077d75756e946945e20efb96a0f6f7530766eb/invenio_rdm_records/assets/semantic-ui/js/invenio_rdm_records/src/deposit/api/DepositApiClient.js#L11-L18).